doc/source/tutorial/interpolate/1D.rst

```diff
@@ -0,0 +1,332 @@
+where linear pieces join. To produce a smoother curve, you can use cubic
+splines, where the interpolating curve is made of cubic pieces with matching
+first and second derivatives. In code, these objects are represented via the
+``CubicSpline`` class instances. An instance is constructed with the ``x`` and
+``y`` arrays of data, and then it can be evaluated using the target ``xnew``
+values:
+
+    >>> from scipy.interpolate import CubicSpline
+    >>> spl = CubicSpline([1, 2, 3, 4, 5, 6], [1, 4, 8, 16, 25, 36])
+    >>> spl(2.5)
+    5.57
+
+A `CubicSpline` object's ``__call__`` method accepts both scalar values and
+arrays. It also accepts a second argument, ``nu``, to evaluate the 
+derivative of order ``nu``. As an example, we plot the derivatives of a spline:
+
+.. plot::
+
+    >>> from scipy.interpolate import CubicSpline
+    >>> x = np.linspace(0, 10, num=11)
+    >>> y = np.cos(-x**2 / 9.)
+    >>> spl = CubicSpline(x, y)
+
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots(4, 1, figsize=(5, 7))
+    >>> xnew = np.linspace(0, 10, num=1001)
+    >>> ax[0].plot(xnew, spl(xnew))
+    >>> ax[0].plot(x, y, 'o', label='data')
+    >>> ax[1].plot(xnew, spl(xnew, nu=1), '--', label='1st derivative')
+    >>> ax[2].plot(xnew, spl(xnew, nu=2), '--', label='2nd derivative')
+    >>> ax[3].plot(xnew, spl(xnew, nu=3), '--', label='3rd derivative')
+    >>> for j in range(4):
+    ...     ax[j].legend(loc='best')
+    >>> plt.tight_layout()
+    >>> plt.show()
+
+Note that the first and second derivatives are continuous by construction, and
+the third derivative jumps at data points. 
+
+
+Monotone interpolants
+=====================
+
+Cubic splines are by construction twice continuously differentiable. This may
+lead to the spline function oscillating and ''overshooting'' in between the
+data points. In these situations, an alternative is to use the so-called
+*monotone* cubic interpolants: these are constructed to be only once
+continuously differentiable, and attempt to preserve the local shape implied
+by the data. `scipy.interpolate` provides two objects of this kind:
+`PchipInterpolator` and `Akima1DInterpolator` . To illustrate, let's consider
+data with an outlier:
+
+.. plot::
+
+    >>> from scipy.interpolate import CubicSpline, PchipInterpolator, Akima1DInterpolator
+    >>> x = np.array([1., 2., 3., 4., 4.5, 5., 6., 7., 8])
+    >>> y = x**2
+    >>> y[4] += 101
+
+    >>> import matplotlib.pyplot as plt
+    >>> xx = np.linspace(1, 8, 51)
+    >>> plt.plot(xx, CubicSpline(x, y)(xx), '--', label='spline')
+    >>> plt.plot(xx, Akima1DInterpolator(x, y)(xx), '-', label='Akima1D')
+    >>> plt.plot(xx, PchipInterpolator(x, y)(xx), '-', label='pchip')
+    >>> plt.plot(x, y, 'o')
+    >>> plt.legend()
+    >>> plt.show()
+
+
+Interpolation with B-splines
+============================
+
+As an illustration, we construct the interpolation of a sine function with
```

Comment:
"As an illustration" of what, though?
There are nice transition sentences at the top of the first few sections. It would be nice to have just one sentence like that to introduce why B-splines are needed.

Reply:
Thanks Matt! Tried to reword the paragraph in the last commit, does it read better?